build/config/compiler/BUILD.gn

```diff
@@ -283,9 +283,12 @@ config("compiler") {
 
     cflags += common_mac_flags
 
-    # Without this, the constructors and destructors of a C++ object inside
-    # an Objective C struct won't be called, which is very bad.
-    cflags_objcc = [ "-fobjc-call-cxx-cdtors" ]
+    cflags_objcc = [
+      "-std=c++11",
```

Comment:
what about the rest of ccflags_cc (like -fno-exceptions, etc)? do we want those too?